nucypher/characters/chaotic.py

```diff
@@ -112,7 +113,7 @@ def __init__(self,
         # Database
         self.db_filepath = db_filepath
         self.db = NO_DATABASE_AVAILABLE
-        self.db_engine = create_engine(f'sqlite:///{self.db_filepath}', convert_unicode=True)
+        self.db_engine = create_engine(f'sqlite:///{self.db_filepath.absolute()}', convert_unicode=True)
```

Comment:
I  cannot seem to find the documentation for this method - Does it return a path with a leading slash?

Reply:
It [creates](https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine) a new `sqlachemy.engine.Engine` instance.

Reply:
Oh, I meant the `absolute` method.

Reply:
Yes. That should give us four leading slashes in total, which is a correct [connection URI](https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/#connection-uri-format).